scripts/build.py

```diff
@@ -105,21 +102,38 @@ def copy_files_source_to_target(source, target):
             shutil.copyfile(source_path, target_path)
 
 
-def _build_files():
-    ensure_directory_exists(OUT_DIR)
-    shutil.rmtree(OUT_DIR)
-    process_third_party_libs()
+def build_files(source, target, ignore=None):
+    """Minifies all css and js files, and removes whitespace from html in source
+    directory and copies it to target, ignoring paths/files mentioned in ignore.
+    Copies files in ignore to target without any changes.
```

Comment:
Is target a string? Is ignore an array of strings? Ditto for source. Mentioning the types is helpful.